src/assets/js/sfu/sfu.js

```diff
@@ -54,6 +54,19 @@ export default class SFU extends EventEmitter {
         const video = this.config.localVideoEl;
         await this.sfuRoom.sendVideo(video.srcObject.getVideoTracks()[0]);
         await this.sfuRoom.sendAudio(video.srcObject.getAudioTracks()[0]);
+        
+        // Attach SoundMeter to Local Stream
+        if (SoundMeter) {
+          // Soundmeter
+          window.stream = video.srcObject;
```

Comment:
Against using window.stream.

But if we want than we can move all magic inside soundmeter and no need to add this to the sfu.js

Reply:
done